Wikipedia/UI-V5/WMFTableOfContentsViewController.swift

```diff
@@ -129,7 +134,15 @@ public class WMFTableOfContentsViewController: UITableViewController, WMFTableOf
         super.viewDidDisappear(animated)
         deselectAllRows()
     }
-
+    
+    public override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
+        
+        coordinator.animateAlongsideTransition({ (context) -> Void in
```

Comment:
@bgerstle this is just how Xcode autocompletes the blocks in the methods. I didn't really argue with it - all the closure stuff in swift reads weird to me.